client/src/styles/index.css

```diff
@@ -0,0 +1,20 @@
+:root{
+  --small-width:30px;
+  --medium-width:25px;
+  --large-width:20px;
+  --small-icon-width:35px;
+  --medium-icon-width:33px;
+  --large-icon-width:30px;
+}
+
+body {
+    background-color: #FDF6F0;
+}
+
+.div-login-attemped {
+  color: red;
+}
+
+.div-signup-alertMessage {
+  color:red;
+}
```

Comment:
These classes appear to only exist on the Login and Signup pages. Could you please move them to respective Login.css and Signup.css files?